internal/manager/manager.go

```diff
@@ -96,8 +96,8 @@ func RunNsmgr(ctx context.Context, configuration *config.Config) error {
 
 	// Context to use for all things started in main
 	m.ctx, m.cancelFunc = context.WithCancel(ctx)
-	m.ctx = log.WithFields(m.ctx, map[string]interface{}{"cmd": "Nsmgr"})
-	m.ctx = log.WithLog(m.ctx, logruslogger.New(m.ctx))
+	logCtx := log.WithLog(m.ctx, logruslogger.New(m.ctx))
+	logCtx = log.WithFields(logCtx, map[string]interface{}{"cmd": "Nsmgr"})
```

Comment:
logger = logruslogger.New(m.ctx)

logger...

Reply:
TODO1: add spanlogger
TODO2: use the logger in each place where we are logging in this code file.

Reply:
ok, removed logrus, added separate context with spanlogger